### PR TITLE
Rename `download` helper

### PR DESCRIPTION
**Pull request by johnnyomair**
The helper is primarily used to create a `FileUploadInput` (previously `FileUploadInterface`) input for `FilesService#upload` while creating fixtures. However, the name of the helper is too generic to be part of the package's public API. We add a new `FileUploadService#createFileUploadInputFromUrl` method to be used instead.

### CODE REVIEW

**Review by nsams — APPROVED** [review]
my hero!

File: packages/api/cms-api/src/dam/files/file-upload.service.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { v4 as uuid } from "uuid";
+
+import { FileUploadInterface } from "./dto/file-upload.interface";
+import { FilesService } from "./files.service";
+
+const pipeline = promisify(stream.pipeline);
+
+@Injectable()
+class FileUploadService {
+    async createUploadFromUrl(url: string): Promise<FileUploadInterface> {
```
Comment (thomasdax98) [inline packages/api/cms-api/src/dam/files/file-upload.service.ts:17]: Why `createUploadFromUrl` and not just `uploadFromUrl`?

Comment (johnnyomair) [reply packages/api/cms-api/src/dam/files/file-upload.service.ts:17]: > (To me createUploadFromUrl implies that the upload is now created but I still have to do a second step to execute the upload and get the actual file)

That's exactly what you have to do, as the actual upload happens in `FilesService#upload`

Comment (johnnyomair) [reply packages/api/cms-api/src/dam/files/file-upload.service.ts:17]: Oh, I misread your concern, sorry. `uploadFromUrl` sounds like "upload this now" to me. It's the same for `createUploadFromUrl`. Maybe find something new?

Comment (johnnyomair) [reply packages/api/cms-api/src/dam/files/file-upload.service.ts:17]: Suggestions:

1. createUploadFromUrl
2. uploadFromUrl
3. createFileToUploadFromUrl
4. createUploadableFromUrl

@nsams @thomasdax98 what do you prefer?

Comment (nsams) [reply packages/api/cms-api/src/dam/files/file-upload.service.ts:17]: createUploadInputFromUrl?

Comment (johnnyomair) [reply packages/api/cms-api/src/dam/files/file-upload.service.ts:17]: @thomasdax98 what do you think about `createUploadInputFromUrl?`?

Maybe we should rename FileUploadInterface as well? Why isn't this called `FileUploadInput`? 🤔

Comment (thomasdax98) [reply packages/api/cms-api/src/dam/files/file-upload.service.ts:17]: Why not rename it to `FileUploadInput` and the method to `createFileUploadInputFromUrl`. That would be coherent in my opinion

Comment (johnnyomair) [reply packages/api/cms-api/src/dam/files/file-upload.service.ts]: Done: 6129691c10202fab55673e61eb42a21c46c0ac6c

**Review by thomasdax98 — APPROVED** [review]

**Review by nsams — APPROVED** [review]

**Pull request merged**